src/core/server/server.api.md

```diff
@@ -97,6 +99,7 @@ export interface CoreSetup {
         registerOnPreAuth: HttpServiceSetup['registerOnPreAuth'];
         registerAuth: HttpServiceSetup['registerAuth'];
         registerOnPostAuth: HttpServiceSetup['registerOnPostAuth'];
+        registerRouter: HttpServiceSetup['registerRouter'];

```

Comment:
frankly, our documentation is not really useful here, we don't provide information what `registerRouter` is. the same for other `HttpServiceSetup` methods. Should we create a separate issue to fix the problem?